runtimes/ios-macos/resource-usage.mdx

```diff
@@ -0,0 +1,18 @@
+---
+title: 'Resource Usage'
+description: ''
+---
+
+When comparing resource usage between Rive and other libraries, you may encounter differences in CPU and memory usage.
+
+An important note is that Rive uses Metal APIs directly over other APIs and frameworks like Core Animation to be able to adjust its usage for best performance with Rive.
+
+<Note>Lottie is an example of a library that uses Core Animation.</Note>
+
+For libraries using Core Animation, logic and rendering is managed in a separate process, the "Render Server" (`backboardd`). In doing so, CPU and memory usage isn't reported by the app process itself, and instead is reported by `backboardd`. 
```

Comment:
How about making this paragraph and the one below it's own section under it's own header: "Libraries using Core Animation"

Reply:
hmm nevermind

Reply:
The commas confused me at first. Maybe instead:

> ...in a separate process called the "Render Server"